src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -86,6 +86,13 @@ public static bool IsCoreTools(this IEnvironment environment)
             return !string.IsNullOrEmpty(environment.GetEnvironmentVariable(CoreToolsEnvironment));
         }
 
+        public static bool IsFunctionsV2CompatibilityMode(this IEnvironment environment)
```

Comment:
nit: Make this consistent with the property name (drop `Functions` or add `Functions` to the property)

Reply:
I've removed `Functions`. It was getting long when you went to `FunctionsCompatibilityModeChangeTokenSource`...